config/config/common-config.yaml

```
@@ -820,6 +820,15 @@ opa:
 
 trivy:
   enabled: true
+  # comma separated list of namespaces (or glob patterns) to be excluded from scanning
+  excludeNamespaces: ""
+  vulnerabilityScannerEnabled: true
+  sbomGenerationEnabled: false
+  configAuditScannerEnabled: true
+  rbacAssessmentScannerEnabled: true
+  infraAssessmentScannerEnabled: true
+  clusterComplianceEnabled: true
+  exposedSecretScannerEnabled: true
```

```suggestion
  clusterComplianceEnabled: true
  configAuditScannerEnabled: true
  exposedSecretScannerEnabled: true
  infraAssessmentScannerEnabled: true
  rbacAssessmentScannerEnabled: true
  vulnerabilityScannerEnabled: true
  sbomGenerationEnabled: false
```
style: Just a little more logical structure in my opinion.